app/src/main/java/com/example/shareyourtrip/Post.java

```diff
@@ -27,6 +27,20 @@ public Post() {
         this.down = null;
     }
 
+    public Post(Post otherPost)
```

Comment:
Whether or not a post is favorited should definitely be a member of this class 'Post'. Otherwise, how will we know to fill the heart on a post? Doing so will simplify the PostAdapter class a TON, because then we wont have to check the current context of the adapter.